libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -612,19 +618,22 @@ void AP_GPS::detect_instance(uint8_t instance)
     // don't build the less common GPS drivers on F1 AP_Periph
 #if !defined(HAL_BUILD_AP_PERIPH) || !defined(STM32F1)
     switch (_type[instance]) {
+#if AP_GPS_SBF_ENABLED
```

Comment:
We can (perhaps in the future?) specify `define AP_GPS_SBF_ENABLED` in `libraries/AP_HAL_ChibiOS/hwdef/f103-GPS/hwdef.dat` instead of checking for periph/stm32f1 here.